payments_service/settings/base.py

```diff
@@ -30,6 +30,8 @@
 # Application definition
 
 INSTALLED_APPS = (
+    'django.contrib.contenttypes',
```

Comment:
without it, I get an error when [subclassing AnonymousUser](https://github.com/mozilla/payments-service/pull/19/files#diff-12b3c0525e0904312102975bd5540176R10). *shrug*